Command/DebugCommand.php

```diff
@@ -22,37 +23,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $r = new \ReflectionProperty($router, 'receivers');
         $r->setAccessible(true);
 
-        $maxWidths = array(
-            'name' => 4,
-            'service' => 7,
-        );
-
-        $rows = array();
-
-        foreach ($r->getValue($router) as $name => $id) {
-            $maxWidths['name'] = max(4, strlen($name));
-            $maxWidths['service'] = max(7, strlen($id));
-
-            $rows[] = array($name, $id);
-        }
-
-        $header = '| <info>%-' . $maxWidths['name'] . '.' . $maxWidths['name'] . 's</info> | <info>%-' . $maxWidths['service'] . '.' . $maxWidths['service'] . 's</info> |';
-        $pattern = '| <comment>%-' . $maxWidths['name'] . '.' . $maxWidths['name'] . 's</comment> | <comment>%-' . $maxWidths['service'] . '.' . $maxWidths['service'] . 's</comment> |';
-
-        $this->writeLineDelimiter($output, $maxWidths);
-
-        $output->writeln(sprintf($header, 'Message', 'Service'));
-
-        $this->writeLineDelimiter($output, $maxWidths);
-
-        foreach ($rows as $row) {
-            $output->writeln(sprintf($pattern, $row[0], $row[1]));
-            $this->writeLineDelimiter($output, $maxWidths);
+        $rows = [];
+        foreach ($r->getValue($router) as $key => $val) {
+            $rows[] = [$key, $val];
         }
-    }
 
-    protected function writeLineDelimiter($output, $maxWidths)
-    {
-        $output->writeln('+' . str_repeat('-', $maxWidths['name'] + 2) . '+' . str_repeat('-', $maxWidths['service'] + 2) . '+');
+        /** @var \Symfony\Component\Console\Helper\TableHelper $helper */
```

Comment:
http://www.phpdoc.org/docs/latest/references/phpdoc/tags/var.html

> You may use the @var tag to document the “Type” of properties, sometimes called class variables.

It is a personal preference. I don't use a fancy IDE for coding, i just use a simple editor (vim). So to me they have no usage other than clottering up the code. Also the `@var` is used wrong, according to the spec.

I just dont like them :D